aspnetcore/mvc/razor-pages/razor-pages-authorization-conventions/sample/Pages/_Layout.cshtml

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>@ViewData["Title"] - AuthorizationSample</title>
+
+    <environment include="Development">
+        <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.css" />
+        <link rel="stylesheet" href="~/css/site.css" />
+    </environment>
+    @*
+        Don't use the ajax.aspnetcdn.com CDN in production.
```

Comment:
😕

Reply:
I agree. This CDN endpoint is in the template, but the current template doesn't call it out. I mention it over in: https://github.com/aspnet/templating/issues/67

I follow the template closely, but this seems very important to mention.

Reply:
I was just confused by that statement. Seems weird we don't want folks to use it.

Reply:
I spoke to @Eilon offline and he claims that statement might be incorrect. Do you want to leave this comment out for now?

Reply:
@Rick-Anderson is on that topic as the author, so he may know why it's there or where it came from originally. https://docs.microsoft.com/en-us/aspnet/ajax/cdn/overview

Reply:
Well ... there's this issue, too: https://github.com/aspnet/Home/issues/1774

Reply:
I just checked all of the scripts and styles in the RP template in use here: :+1: on gzip everywhere. I'll pull the comments here and on the other PR. I'll leave a note for them over in my templating issue that they might want us to remove that line from the topic (or investigate further for gzipping problems on other endpoints).

Reply:
Yeah I've asked @DamianEdwards to look into the CDN support question.